README.md

```diff
@@ -1,16 +1,71 @@
 # Smsl-php-lib
+We are happy to present the SMS API client for PHP for the SMSLeopard SMS platform. Its meant to help simplify intergration on php based applications.
 
-This is the php helper library for the SMSLeopard api.
+## Getting started
+To use the api you need an account with SMSLeopard, its free to make one. Just head over to the [SMSLeopard homepage](https://www.smsleopard.com/). 
 
-Clone the repo.
+## API Keys
+To generate the api keys, you will need to upgrade to an Enterprise account. This is also a free upgrade, we just need you to send an email request to [info@smsleopard.com](mailto:info@smsleopard.com). 
 
-Navigate to repo folder.
+After that, just log in to your dashboard and click the settings link, then the api keys link.
 
-Install dependencies.
-`composer install`
+Once at the api keys page, click add new key. Enter the key name and follow the instructions to get and save you api keys.
 
-Run unit tests
-`./vendor/bin/phpunit`
+## Installing
+The api client is hosted on packagist. You will need to install composer in your project to install it. Once you have installed composer you can use composer require to download the api client.
+``` 
+composer require smsleopard/smsleopard
+```
 
-Next step:
-    - Prepare the package for upload to packagist.
+## Send a message
+### Send a normal message
+To send a normal message 
+```
+$account_id = 'YOUR ACCOUNT ID';
+$account_key   = 'YOUR ACCOUNT KEY';
+$client      = new Client($account_id, $account_key);
+
+$message = 'Test message';
+$sender_id = 'SMS_TEST';
+$recipients = [['number'=> '+2547960000001']];
+
+$response   = $client->send($sender_id, $message, $recipients);
+```
+
+### Send a custom message
+To send a custom message
+```
+$account_id = 'YOUR ACCOUNT ID';
+$account_key   = 'YOUR ACCOUNT KEY';
+$client      = new Client($account_id, $account_key);
+
+$message = 'Test message {name}';
+$sender_id = 'SMS_TEST';
+$recipients = [
+  ['number'=> '+2547960000001', 'message'=> 'Test message Brian'],
+  ['number'=> '+2547960000002', 'message'=> 'Test message Emma']
+];
+
+$response   = $client->send($sender_id, $message, $recipients, true);
```

Comment:
Lets also add a way to print out this recipients, response status, etc. That will help